Sources/SwiftSyntax/Utils.swift

```diff
@@ -102,3 +102,29 @@ extension RawUnexpectedNodesSyntax {
     self.init(raw: raw)
   }
 }
+
+extension TypeSyntax {
+  public var isVoid: Bool {
```

Comment:
Please provide documentation comments for all public APIs. Once we settle on them, please add an entry to the [release notes](https://github.com/swiftlang/swift-syntax/blob/main/Release%20Notes/601.md).